jerry-core/api/jerry.c

```diff
@@ -727,14 +725,16 @@ bool
 jerry_value_is_promise (const jerry_value_t value) /**< api value */
 {
   jerry_assert_api_available ();
-
+#ifndef CONFIG_DISABLE_ES2015_PROMISE_BUILTIN
   jerry_value_t promise = jerry_get_arg_value (value);
   return (ecma_is_value_object (promise)
           && ecma_is_promise (ecma_get_object_from_value (promise)));
+#else /* !CONFIG_DISABLE_ES2015_PROMISE_BUILTIN */
+  JERRY_UNUSED (value);
+  return false;
+#endif /* CONFIG_DISABLE_ES2015_PROMISE_BUILTIN */
```

Comment:
Just spotted that the preproc else/endif comments have been swapped. I think the endif should have the !CONFIG_DISABLE comment. Could you please fix this too before landing? Sorry for the late hit.